test/specs/dom/index.js

```diff
@@ -0,0 +1,545 @@
+/** only run DOM tests in browsers */
+if (typeof window !== "undefined") {
+  const PalindromDOM = require("../../../src/palindrom-dom");
+  const assert = require("assert");
+  const moxios = require("moxios");
+  const sinon = require("sinon");
+  const expect = require("chai").expect;
+
+  function createLinkTest(href, parent) {
+    parent = parent || document.body;
+    const a = document.createElement("A");
+    a.innerHTML = "Link";
+    a.href = href;
+    parent.appendChild(a);
+    parent.addEventListener("click", clickHandler);
+    fireEvent(a, "click");
+    parent.removeEventListener("click", clickHandler);
+    parent.removeChild(a);
+  }
+  function createLinkTestNested(href, parent) {
+    parent = parent || document.body;
+    const a = document.createElement("A");
+    a.innerHTML = "<strong>Link</strong>";
+    a.href = href;
+    parent.appendChild(a);
+    parent.addEventListener("click", clickHandler);
+    fireEvent(a.firstChild, "click");
+    parent.removeEventListener("click", clickHandler);
+    parent.removeChild(a);
+  }
+
+  function fireEvent(fireOnThis, evt) {
```

Comment:
Sure, [we simulate link clicks with it](https://github.com/Palindrom/Palindrom/blob/70335b8c4de8bf02e37240231b5e2ee78974a6af/test/specs/dom/index.js#L16).